.builds/freebsd.yml

```diff
@@ -35,9 +35,6 @@ tasks:
 - unittest: |
     cd neovim
     gmake unittest
-- oldtest: |
-    cd neovim
-    gmake oldtest
```

Comment:
I've ran it manually from a build with a shell, and this is the result:
```
Executed:  1437 Tests
 Skipped:    52 Tests
  FAILED:     2 Tests


Failures:
        From test_writefile.vim:
        Found errors in Test_writefile_fails_conversion():
        function RunTheTest[37]..Test_writefile_fails_conversion line 12: command did not fail: write ++enc=cp932
        function RunTheTest[37]..Test_writefile_fails_conversion line 13: Expected ['line one', 'line two'] but got ['first line', 'cannot convert ', 'third line']
        From test_alot.vim:
        Found errors in Test_libcall_libcallnr():
        Caught exception in Test_libcall_libcallnr(): Vim(call):dlerror = "/usr/local/lib/: read error: Invalid argument" @ function RunTheTest[37]..Test_libcall_libcallnr, line 32

TEST FAILURE
gmake[1]: *** [Makefile:94: report] Error 1
gmake[1]: Leaving directory '/usr/home/build/neovim/src/nvim/testdir'
gmake: *** [Makefile:119: oldtest] Error 2
[build@build /usr/home/build/neovim]$ uname -a
FreeBSD build 12.1-RELEASE-p1 FreeBSD 12.1-RELEASE-p1 GENERIC  amd64
```

For some reason (Sourcehut bug?) it does not generate a log for it apparently still though.